simulator/src/listeners.js

```diff
@@ -1113,13 +1113,14 @@ function zoomSliderListeners() {
     // here lenght-2 is done because last two button are used for diff purpose
     for(var quickmenui = 0; quickmenui < quickMenu.length - 2; quickmenui++) {
         (function(index) {
-            quickMenu[index].addEventListener('click', (e) => {
+            function clickOrtouch(e) {
                 onTapColor(quickMenu, index, buttoncolor);
                 onQuickmenuTap(index);
                 setTimeout(() => {
                     onTapColor(quickMenu, index, ''); }, 100);
                 e.preventDefault();
-            });
+            }
+            quickMenu[index].addEventListener('touchend', (e) => clickOrtouch(e))
```

Comment:
@pulkit-30 should I remove the parentheses around the (e) ? isn't it better to keep the parentheses as it is easier to identify to arrow function because of it

Reply:
@Arnabdaz Yes, read more about this [here](https://eslint.org/docs/latest/rules/arrow-parens)

Reply:
@Arnabdaz Great work.
Could you try just replacing the 'click'  with 'touchend'? Maybe it will fix the code climate issue. I think the issue is because the code climate is expecting curly braces after an arrow function.

Reply:
@Prerna-0202 actually I tried that first but due to some reason it was not working after multiple tries but when I made a separate function it worked without any issues. So I thought to keep it that way.
anyways I fixed the code climate issue now please take a look.